boards/arm/mimxrt1064_evk/mimxrt1064_evk.dts

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2018, NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <nxp/nxp_rt.dtsi>
+
+/ {
+	model = "NXP MIMXRT1064-EVK board";
+	compatible = "nxp,mimxrt1064";
+
+	aliases {
+		gpio-1= &gpio1;
+		gpio-2= &gpio2;
+		gpio-3= &gpio3;
+		gpio-4= &gpio4;
+		gpio-5= &gpio5;
+		uart-1 = &uart1;
+		led0 = &green_led;
+		sw0 = &user_button;
+	};
+
+	chosen {
+#if defined(CONFIG_CODE_ITCM)
+		zephyr,flash = &itcm0;
+#elif defined(CONFIG_CODE_INTERNAL_QSPI)
+		zephyr,flash = &qspi1;
+#endif
+		zephyr,sram = &dtcm0;
+		zephyr,console = &uart1;
+	};
+
+	sdram0: memory@80000000 {
+		/* Micron MT48LC16M16A2B4-6AIT:G */
+		device_type = "memory";
+		reg = <0x80000000 0x2000000>;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		green_led: led-1 {
+			gpios = <&gpio1 9 GPIO_INT_ACTIVE_LOW>;
+			label = "User LD1";
+		};
+	};
+
+	gpio_keys {
+		compatible = "gpio-keys";
+		user_button: button-1 {
+			label = "User SW8";
+			gpios = <&gpio5 0 GPIO_INT_ACTIVE_LOW>;
+		};
+	};
+};
+
+&flexspi0 {
+	qspi1: qspi@70000000 {
+		/* WINBOND */
```

Comment:
Why windbond?  From schematics looks like an ISSI part?

Reply:
since it is using the internal flash die from Winbond